WebAPI/Controllers/StudentController.cs

```diff
@@ -180,5 +179,61 @@ public async Task<IActionResult> AddNewBlock([FromBody]AddNewBlockModel newBlock
             }
             return Ok();
         }
+
+        [HttpPut("editBlock")]
+        public async Task<IActionResult> EditBlock([FromBody]AddNewBlockModel newBlockModel)
```

Comment:
Question is (check other controllers) if this is not too much logic for controller...
If create course... shouldn't be part of the model...